apps/src/templates/HintsDisplay.jsx

```diff
@@ -42,8 +68,15 @@ module.exports = React.createClass({
       seenHints = [
           <h1>{ this.props.hintReviewTitle }</h1>,
           <ul>
-            {hintsToShow.map(function (hint) {
-              return <li dangerouslySetInnerHTML={{ __html : hint.content }} style={{ marginBottom: '12px' }}/>;
+            {hintsToShow.map(function (hint, i) {
+              var hintBlock;
+              if (hint.block) {
+                hintBlock = (<div className="block-hint" id={ hint.hintId } style={{ maxHeight: '100px' }} />);
+              }
+              return (<li style={{ marginBottom: '12px' }}>
+                <div dangerouslySetInnerHTML={{ __html : hint.content }} />
+                {hintBlock}
+              </li>);
             })}
```

Comment:
Looks like you added `i` to the map function, but don't actually use it?

I might extract this to a super small Hint component, so that you can just do something like
```
hintsToShow.map(function (hint, i) {
  return <Hint hint={hint}/>
});
```